src/mishmash.ts

```diff
@@ -0,0 +1,27 @@
+import { Storage } from './utils'
+
+export const showRecipes = (selectedIngredients: string[]) => {
+    if (localStorage.getItem('recipes') === null) {
+        return
+    }
+    
+    let result: Object
+    const recipesIndex: number = 1
+    const receivedRecipes: Array<Object> = Storage.get('recipes')
+
+    result = receivedRecipes.find((recipe: Object) => {
+        const arr: Array<[]> = Object.values(recipe)[recipesIndex]
+
+        return arr.flat().every((ingredient: string) => selectedIngredients.includes(ingredient) && arr.flat().length === selectedIngredients.length)
```

Comment:
typescript will know that ingredient is a string if `arr` will be typed correctly. Also `arr` is a bad name